### DISCUSSION TITLE
Use helm for hcloud-cloud-controller-manager

### DISCUSSION THREAD

**Discussion by olexiyb** [mysticaltech/terraform-hcloud-kube-hetzner#1429] [Ideas]
According to [readme](https://github.com/hetznercloud/hcloud-cloud-controller-manager/blob/main/README.md) helm is recommended way to install it
Also as part of my [pull](https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/pull/1405)
I have used 
[doc](https://github.com/hetznercloud/hcloud-cloud-controller-manager/blob/main/docs/robot.md)
What advantages give direct deployment?
Helm is just better way to manage any deployment

**Comment by mysticaltech** [comment]
This is done! Thanks @olexiyb
